content/ngf/how-to/scaling.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Scaling the control plane and data plane
+weight: 700
+toc: true
+type: how-to
+product: NGF
+docs: DOCS-0000
+---
+
+This document describes how you can separately scale the NGINX Gateway Fabric control plane and data plane.
+
+It provides guidance on how to scale each plane effectively, and when you should do so based on your traffic patterns.
+
+
+### Scaling the data plane
+
+The data plane is the NGINX deployment that handles user traffic to backend applications. Every Gateway object created provisions its own NGINX deployment and configuration. 
+
+You have two options for scaling the data plane:
+
+- Increasing the number of replicas for an existing deployment
+- Creating a new Gateway for a new data plane
+
+#### When to increase replicas or create a new Gateway
+
+Understanding when to increase replicas or create a new Gateway is key to managing traffic effectively.
+
+Increasing data plane replicas is ideal when you need to handle more traffic without changing the configuration. 
+
+For example, if you're routing traffic to `api.example.com` and notice an increase in load, you can scale the replicas from 1 to 5 to better distribute the traffic and reduce latency. 
+
+All replicas will share the same configuration from the Gateway used to set up the data plane, simplifying configuration management.
+
+There are two ways to modify the number of replicas for an NGINX deployment:
+
+First, by modifying the field `nginx.replicas` in the `values.yaml` or adding the `--set nginx.replicas=` flag to the `helm install` command.
```

Comment:
I'd be clear that this is at installation time, and the second option is after nginx is already running.